### ISSUE TITLE
NTR: Corynebacterium diphtheriae present on cultures (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8099]
New term request Corynebacterium diphtheriae present on cultures (nadea3_190423203740): Nasopharyngeal cultures; special cultures include cysteine-tellurite blood agar or modified Tinsdale agar  UMLS: 2238155

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000742
- label: Positive nasopharyngeal corynebacterium diphtheriae nucleic acid test
- parents: [Positive oropharangeal infectious agent test (HP:6000579)](https://hpo.jax.org/app/browse/term/HP:6000579)
- pmid: 
- def: Detection of nucleic acid of Corynebacterium diphtheriae in nasopharyngeal secretions by a method such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8099
- synonyms:

**Comment by MickeySegal** [comment]
This was supposed to be cultures.
